pkg/apis/pipeline/v1alpha1/pipeline_validation_test.go

```diff
@@ -310,7 +310,7 @@ func TestPipeline_Validate(t *testing.T) {
 		p: tb.Pipeline("name", "namespace", tb.PipelineSpec(
 			tb.PipelineWorkspaceDeclaration("foo"),
 			tb.PipelineTask("taskname", "taskref",
-				tb.PipelineTaskWorkspaceBinding("taskWorkspaceName", "pipelineWorkspaceName")),
+				tb.PipelineTaskWorkspaceBinding("taskWorkspaceName", tb.PipelineTaskWorkspaceBindingWorkspace("pipelineWorkspaceName"))),
```

Comment:
i think maybe there should be some more unit test cases added here? or are they caught somewhere else? (i remember the validation unit tests being a bit hard to puzzle out what is tested where...)